### DISCUSSION TITLE
MAUIのフォローアイコン

### DISCUSSION THREAD

**Discussion by sekinekh** [increments/qiita-discussions#937] [要望・アイデア・不具合]
MAUIの記事をフォローしていますが、フォローアイコンのイメージが正しく読み込まれません。
気持ちが悪いので直していただきたいです。

**Comment by shiracamus** [comment]
タグページでアイコンをクリックするとタグアイコン編集申請画面になって、アイコン画像をアップロードして編集依頼できますよ。

**Comment by tomoasleep** [comment]
ご不便をおかけしております。 (@shiracamus さんはタグアイコン申請についてご説明いただきありがとうございます :bow: )

MAUI タグには、タグアイコン画像が設定されておらず、こういった場合は代替画像を表示していたのですが、不具合で代替画像がちゃんと表示出来ていない状態になっております。この不具合は修正予定です :bow:
MAUI タグに関しては、アイコン画像の申請があり承認いたしましたので、現在はアイコン画像が表示されるようになっております。

また、@shiracamus さんが説明してくださった通り、タグのアイコン画像は申請を行うことが可能です。

以下の手順で申請することが出来、運営側で確認の後、問題がなければ反映されます。
https://help.qiita.com/ja/articles/qiita-tag#part-2dc30e68f1f0c4db 

アイコン画像が設定されていないタグに関しては、申請いただけるととても嬉しいです 😃

**Reply by nekohei** [reply]
@tomoasleep 
アイコンが反映されていることを確認しました。
ご対応ありがとうございます。

@shiracamus 
お陰様でご対応いただきました。ありがとうございました！

**Comment by sekinekh** [comment]
@tomoasleep
アイコンが反映されていることを確認しました。
ご対応ありがとうございます。

@shiracamus
お陰様でご対応いただきました。ありがとうございました！
